### ISSUE TITLE
Feature Request: add option to remove printing of control characters (--no-control-characters ?)

### ISSUE THREAD

**Issue by NewtonsNerds** [palantir/tslint#3470]
### Bug Report (Feature Request)

- __TSLint version__: 5.8.0
- __TypeScript version__: 2.5.3
- __Running TSLint via__: CLI (yarn)

#### TypeScript code being linted

```ts
// Not actually relevant?  Any code will do.  I will edit if it is deemed necessary.
```

with `tslint.json` configuration:

```json
{
  "extends": [
    "tslint-config-standard",
    "tslint-immutable",
    "tslint-react"
  ],
  "rules": {
    "jsx-no-multiline-js": false,
    "jsx-boolean-value": false,
    "typedef": [ true, "call-signature" ],
    "no-consecutive-blank-lines": false,
    "no-let": true,
    "no-var-keyword": true
  }
}
```

#### Actual behavior
Depending on the terminal, the control characters print:
```
 RUNS  ...
[1A
 RUNS  ...
[1A
 RUNS  ...
[1A
 RUNS  ...
[1APASS src/renderer/foo.tsx
```

#### Expected behavior
With this option (--no-control-characters ?):
```
 RUNS  ...
PASS src/renderer/foo.tsx
```

**Comment by ajafff** [comment]
What formatter are you using? This doesn't look like any of the core formatters.

What's your use case for not wanting any control characters in the output?

**Comment by NewtonsNerds** [comment]
*facepalm*

Sorry, this is actually meant to be for jest.  I just always `yarn lint && yarn test` so I confused myself.
